testdata/book/httpbin.yml

```diff
@@ -33,28 +36,60 @@ steps:
         post:
           body:
             application/json: "{{ vars.jsonRequestBody }}"
-  -
     test: |
-      steps[2].res.status == 200
-      && steps[2].res.body.json == vars.jsonRequestBody
+      steps[1].res.status == 200
+      && steps[1].res.body.json == vars.jsonRequestBody
   -
     req:
       /put:
         put:
           body:
             application/json: "{{ vars.arrayJsonRequestBody }}"
-  -
     test: |
-      steps[4].res.status == 200
-      && steps[4].res.body.json == vars.arrayJsonRequestBody
+      steps[2].res.status == 200
+      && steps[2].res.body.json == vars.arrayJsonRequestBody
   -
     req:
       /patch:
         patch:
           body:
             application/json: "{{ vars.external }}"
-  -
     test: |
-      steps[6].res.status == 200
-      && steps[6].res.body.json == vars.external
+      steps[3].res.status == 200
+      && steps[3].res.body.json == vars.external
       && vars.external == vars.jsonRequestBody
+  -
+    loop: vars.loopCount
+    req:
+      /delete:
+        delete:
+          body:
+            application/json:
+              count: "{{ i }}"
+    test: |
+      steps[4].res.status == 200
+      && steps[4].res.body.json.count == i
+  -
+    loop:
+      count: 5
+      until: 'steps[5].res.status != "200"'
+      minInterval: 0.5
+      maxInterval: 10
+    req:
+      /status/{{ 204 - i }}:
+        get:
+          body: null
+    test: |
+      steps[5].res.status == (204 - i)
+  -
+    test: |
+      steps[5].res.status == 204
+    # FIXME: I'd like to see the last status be 200.
```

Comment:
@k1LoW 

What is the best rule for referencing the result of a loop process?

* Keep only the final result.
Currently, I think the first result is referenced.
* Allow reference to the result of each loop.
`steps[5].res[4].status`.

Reply:
Maybe the until: condition is incorrect.

```diff
- until: 'steps[5].res.status != "200"'
+ until: 'steps[5].res.status == "200"'
```

Reply:
Fixed 👍 https://github.com/k1LoW/runn/pull/97/commits/c26f27c2243aafaaa48dea72387aacb643553615